src/ParallelHashMap.h

```diff
@@ -820,19 +860,20 @@ void ParallelHashMap<K,V>::clear()
  *      not freed during access.
  */
 template <class K, class V>
-void ParallelHashMap<K,V>::print_buckets()
-{
+void ParallelHashMap<K,V>::print_buckets() {
+
   /* get thread ID */
   size_t tid = 0;
-  #ifdef OPENMP
+#ifdef OPENMP
   tid = omp_get_thread_num();
-  #endif
+#endif
 
   /* get pointer to table, announce it will be searched */
   FixedHashMap<K,V> *table_ptr;
-  do{
+  do {
     table_ptr = _table;
     _announce[tid].value = table_ptr;
+#pragma omp flush(_announce)
```

Comment:
Right. Just the ``flushes``, an updated ``at(K key)``, and a new ``update(K key, V value)`` function